lib/trainer/test_parser.rb

```diff
@@ -108,18 +108,25 @@ def unfold_tests(data)
     end
 
     # Convert the Hashes and Arrays in something more useful
-    def parse_content
+    def parse_content(xcpretty_naming)
       self.data = self.raw_json["TestableSummaries"].collect do |testable_summary|
         summary_row = {
           project_path: testable_summary["ProjectPath"],
           target_name: testable_summary["TargetName"],
           test_name: testable_summary["TestName"],
           duration: testable_summary["Tests"].map { |current_test| current_test["Duration"] }.inject(:+),
           tests: unfold_tests(testable_summary["Tests"]).collect do |current_test|
+			if xcpretty_naming
+				test_group = testable_summary["TargetName"] + "." + current_test["TestIdentifier"].split("/")[0..-2].join(".")
+				test_name = current_test["TestName"][0..-3]
+			else
+				test_group = current_test["TestIdentifier"].split("/")[0..-2].join(".")
+				test_name = current_test["TestName"]
+			end
             current_row = {
               identifier: current_test["TestIdentifier"],
-			  test_group: testable_summary["TargetName"] + "." + current_test["TestIdentifier"].split("/")[0..-2].join("."),
-			  name: current_test["TestName"][0..-3],
+			  test_group: test_group,
```

Comment:
Rubocop should fix these whitespace issues :)

Reply:
Thanks for the command ! Fixed it.